City-of-Helsinki/drupal-helfi-platform-config
# UHF-2524: Errand service layout changes

How to test:
* If you don't already have a site for testing, create one:
  * `composer create-project City-of-Helsinki/drupal-helfi-platform:dev-main hel-platform --no-interaction --repository https://repository.drupal.hel.ninja/`
  * `cd hel-platform`
* Checkout correct branches: `composer require drupal/helfi_platform_config:dev-UHF-2524_errand_service_layout_changes && composer require drupal/hdbt:dev-UHF-2524_errand_service_layout_changes`
* `make new`
* `make shell`
* Enable TPR module with configuration `drush en -y helfi_tpr_config`
* Import services `drush mim tpr_service`
* Import errand servives `drush mim tpr_errand_service`
* Import service channels `drush mim tpr_service_channel`
* `drush cr`
* `drush uli` and log into the site using the link.
* Go to a service page that has plenty of errand services to make sure the layout works nicely. Good example is `/fi/tpr-service/3641/`
* The service page should now have errands services built so that first there is the errand service name and description, after that a accordion that has details of the service and as last element there is the service channels of the errand service if any. Check the layout from here: https://share.goabstract.com/2d4ad388-7753-4756-ba1d-6c6cbde6e425?collectionLayerId=3e29b4ca-53ff-49ab-a0a2-afc08a07270e&mode=build&selected=root-CEC4000B-8D0E-4EDE-AC72-7D780AC092E0 The implemented version is the "Versio 2".
* The service channel layout looks old still and those are fixed in a separate story. Here you should concentrate on the structure more than the layout being 1:1.

Bonus task: Test this feature with some instance as well but not Sote - Sote does not have the errand service information visible.

Also check these:
https://github.com/City-of-Helsinki/drupal-hdbt/pull/150
